specs/schema.task_execution_service.d55bf88.openapi.modified.yaml

```diff
@@ -600,15 +616,20 @@ definitions:
     readOnly: true
   tesTasksInfo:
     properties:
-      id:
-        type: string
-        description: ''
-      execution_time:
-        type: string
-        description: ''
+      cost:
+        $ref: '#/definitions/tesCost'
       queuing_time:
+        type: number
+        format: int64
+        description: 'Time required for the running of the'
```

Comment:
Time in which unit, seconds?

Reply:
queuing_time_unit parameter should cover that .. I didn't want to add a new model .. not sure why it doesn't show up here ... will just update